freshsales-analytics.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.6"
+  spec.add_development_dependency "bundler", "~> 2.2.10"
```

Comment:
Can we changes this to "~> 2.2" so that the latest bundle version will be support, if 2.2.10 is specified it will support only till 2.2.X?

https://rubygems.org/gems/bundler/versions